fdbserver/ptxn/test/Driver.actor.cpp

```diff
@@ -109,6 +114,24 @@ void startFakeProxy(std::vector<Future<Void>>& actors, std::shared_ptr<TestDrive
 	}
 }
 
+// Starts all fake resolvers. For now, use "resolverCore" to start the actor.
+// TODO: change to "resolver" after we have fake ServerDBInfo object.
+void startFakeResolver(std::vector<Future<Void>>& actors, std::shared_ptr<TestDriverContext> pTestDriverContext) {
+	for (int i = 0; i < pTestDriverContext->numResolvers; ++i) {
+		ResolverInterface recruited;
+		// recruited.locality = locality;
+		recruited.initEndpoints();
+
+		InitializeResolverRequest req;
+		req.recoveryCount = 1;
+		req.commitProxyCount = pTestDriverContext->numProxies;
+		req.resolverCount = pTestDriverContext->numResolvers;
+
+		Future<Void> core = ::resolverCore(recruited, req);
```

Comment:
I am uncertain if this `resolverCore` available?

Reply:
Why won't the linker find a definition?

Reply:
This compiles fine, i.e., the linker finds the definition of the function.

Reply:
> Why won't the linker find a definition?

I thought this was for the new Resolver. I believe this `resolverCore` is the original resolver?